samples/matter/common/src/bridge/Kconfig

```diff
@@ -76,4 +78,8 @@ config BRIDGE_BT_CONNECTION_LATENCY
 	int "It allows the peripheral to skip waking up for a certain number of connection events if it does not have any data to send"
 	default 0
 
+config BRIDGE_SET_BT_CONNECTION_PARAMS
```

Comment:
I think that "force" instead of "set" better explains what we try to achieve here.